resource-managers/kubernetes/docker/src/main/dockerfiles/spark/entrypoint.sh

```diff
@@ -110,8 +110,7 @@ case "$SPARK_K8S_CMD" in
     ;;
 
   *)
-    echo "Unknown command: $SPARK_K8S_CMD" 1>&2
-    exit 1
+    CMD=("$@")
```

Comment:
Should log a message here too, about "executing in pass-through mode" since this is the guaranteed code path for pass through